engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/ConfigFactoryExt.scala

```diff
@@ -4,8 +4,12 @@ import com.typesafe.config.{Config, ConfigFactory}
 
 import java.io.File
 import java.net.URI
+import scala.util.Try
 
 object ConfigFactoryExt {
+
+  private val configLocationProperty = "config.locations"
```

Comment:
1. Maybe we should do it public?
2. Maybe it should be "nk.config.locations". To clarify that it is our location